priv/templates/nimble_template/test/support/feature_case.ex.eex

```diff
@@ -10,6 +10,8 @@ defmodule <%= web_module %>.FeatureCase do
       use Mimic
       use <%= web_module %>, :verified_routes
 
+      use <%= web_module %>, :verified_routes
```

Comment:
seem we have the `use <%= web_module %>, :verified_routes` above @liamstevens111, I have added it on my PR :D 
```suggestion
```